server/routes/paths/ppcs.ts

```diff
@@ -4,6 +4,7 @@ export const ppcsPaths = {
   consecutiveSentenceDetails: 'consecutive-sentence-details',
   matchIndexOffence: 'match-index-offence',
   determinatePpudSentences: 'determinate-ppud-sentences',
+  editCustodyType: 'custody-type',
```

Comment:
There are a few places in other sources you've changed in this PR where this same string is used - could you change those to refer to this value here instead, please?